.evergreen/install-tools.sh

```diff
@@ -39,10 +39,10 @@ swiftenv local $SWIFT_VERSION
 
 if [ $1 == "swiftlint" ]
 then
-    build_from_gh swiftlint https://github.com/realm/SwiftLint "0.44.0"
+    build_from_gh swiftlint https://github.com/realm/SwiftLint "0.46.2"
```

Comment:
looks like `0.46.3` was just [released](https://github.com/realm/SwiftLint/releases/tag/0.46.3) yesterday, do we want to use that version instead? (ditto other PR)

Reply:
good catch, I was just looking at the latest version that homebrew had which lags a bit. updated